app/build.gradle.kts

```diff
@@ -54,17 +55,27 @@ android {
         versionName = "4.1.2"
 
         resValue("string", "app_version", "${defaultConfig.versionName}${versionNameSuffix ?: ""}")
-
         resValue("string", "commit_hash", "git rev-parse --short HEAD".execute() ?: "")
-
         resValue("bool", "is_prerelease", "false")
 
+        // Reads local.properties
+        val localProperties = gradleLocalProperties(rootDir)
+
         buildConfigField(
             "String",
             "BUILDDATE",
             "new java.text.SimpleDateFormat(\"yyyy-MM-dd HH:mm\").format(new java.util.Date(" + System.currentTimeMillis() + "L));"
         )
-
+        buildConfigField(
+            "String",
+            "SIMKL_CLIENT_ID",
+            "\"" + (System.getenv("SIMKL_CLIENT_ID") ?: localProperties["simkl.id"]) + "\""
```

Comment:
could you add these env vars to `.github/workflows/build_to_archive.yml` and `.github/workflows/prerelease.yml`?

Reply:
Added but untested :+1: